json/Explain_Potential.txt

```diff
@@ -9767,175 +9767,175 @@
 	{
 		"assign": "1919631745",
 		"jp_text": "未設定潜在能力",
-		"tr_text": "",
+		"tr_text": "Ability not set",
 		"jp_explain": "未設定潜在能力",
-		"tr_explain": ""
+		"tr_explain": "Ability not set"
 	},
 	{
 		"assign": "1919631746",
 		"jp_text": "未設定潜在能力",
-		"tr_text": "",
+		"tr_text": "Ability not set",
 		"jp_explain": "未設定潜在能力",
-		"tr_explain": ""
+		"tr_explain": "Ability not set"
 	},
 	{
 		"assign": "1919631747",
 		"jp_text": "未設定潜在能力",
-		"tr_text": "",
+		"tr_text": "Ability not set",
 		"jp_explain": "未設定潜在能力",
-		"tr_explain": ""
+		"tr_explain": "Ability not set"
 	},
 	{
 		"assign": "1919631772",
 		"jp_text": "未設定潜在能力",
-		"tr_text": "",
+		"tr_text": "Ability not set",
 		"jp_explain": "未設定潜在能力",
-		"tr_explain": ""
+		"tr_explain": "Ability not set"
 	},
 	{
 		"assign": "1919631773",
 		"jp_text": "未設定潜在能力",
-		"tr_text": "",
+		"tr_text": "Ability not set",
 		"jp_explain": "未設定潜在能力",
-		"tr_explain": ""
+		"tr_explain": "Ability not set"
 	},
 	{
 		"assign": "1919539126",
 		"jp_text": "閃梗華 Lv.1",
-		"tr_text": "",
+		"tr_text": "Blossoming End Lv.1",
 		"jp_explain": "カンランキキョウの\n威力を上昇させる。",
 		"tr_explain": ""
 	},
 	{
 		"assign": "1919539127",
 		"jp_text": "閃梗華 Lv.2",
-		"tr_text": "",
+		"tr_text": "Blossoming End Lv.2",
 		"jp_explain": "カンランキキョウの\n威力を上昇させる。",
 		"tr_explain": ""
 	},
 	{
 		"assign": "1919539124",
 		"jp_text": "閃梗華 Lv.3",
-		"tr_text": "",
+		"tr_text": "Blossoming End Lv.3",
 		"jp_explain": "カンランキキョウの\n威力を上昇させる。",
 		"tr_explain": ""
 	},
 	{
 		"assign": "1919539125",
 		"jp_text": "閃梗華 Lv.4",
-		"tr_text": "",
+		"tr_text": "Blossoming End Lv.4",
 		"jp_explain": "カンランキキョウの\n威力を上昇させる。",
 		"tr_explain": ""
 	},
 	{
 		"assign": "1919538250",
 		"jp_text": "閃梗華 Lv.5",
-		"tr_text": "",
+		"tr_text": Blossoming End Lv.5"",
```

Comment:
Copy and paste issue here? the text is not in the quotes

Reply:
ughhh I'll get it